src/app/quiz/components/Question.js

```diff
@@ -40,6 +55,11 @@ function BoxHeader({ text }) {
       sx={{
         fontWeight: "bold",
         textAlign: "center",
+        fontSize: {
```

Comment:
Generally avoid setting custom font sizes to fonts. That contradicts MUI typography system, which predefines font styles and sizes.
https://mui.com/material-ui/react-typography/#usage

Reply:
I returned it as it was, but it looks a little ugly on mobile (some inscriptions do not fit on the screen)

Before new commit (with changes):
![image](https://github.com/digital-brother/wct-fe/assets/49399157/4418be75-f7c5-4cd0-99da-5bb8a46ed896)
![image](https://github.com/digital-brother/wct-fe/assets/49399157/cc507ad2-b83e-4dad-a69a-76e53e476eaa)
![image](https://github.com/digital-brother/wct-fe/assets/49399157/b0098b0c-69b2-4377-828c-ef3f824a8a20)
![image](https://github.com/digital-brother/wct-fe/assets/49399157/0ce3aff7-33cd-49e3-85c4-d2fc1a21e0c6)

After:
![image](https://github.com/digital-brother/wct-fe/assets/49399157/5f460514-c4e0-4733-9d6f-c0a6d677dda0)
![image](https://github.com/digital-brother/wct-fe/assets/49399157/a553f075-ddd8-40b4-ac1f-c71b8169434e)
![image](https://github.com/digital-brother/wct-fe/assets/49399157/7f58ff3c-1655-4b72-b970-1105a331ba97)
![image](https://github.com/digital-brother/wct-fe/assets/49399157/003249c2-45fb-4704-a5e1-b26b1d158eb0)

![image](https://github.com/digital-brother/wct-fe/assets/49399157/c9b0ca4b-b643-4228-9146-6f3ca13715ed)
![image](https://github.com/digital-brother/wct-fe/assets/49399157/9ca77bef-d08b-4e56-812f-a9b372d34577)